src/array.cr

```diff
@@ -1023,6 +1023,7 @@ class Array(T)
   # :nodoc:
   protected def size=(size : Int)
     @size = size.to_i
+    size
```

Comment:
This change is wrong. The `size` argument and the `Array#size` getter may have different types.

Reply:
There's no need for them to be the same. Setters ought to return the given value, not the one actually returned by the corresponding getter.